### PR TITLE
elastix: 4.9.0 -> 5.0.0 (and mark unbroken)

### PR DESCRIPTION
**Pull request by bcdarwin**
###### Motivation for this change

For ZHF (#68361).

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [NA] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [NA] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/development/libraries/science/biology/elastix/default.nix [hunk 1]
```diff
@@ -2,12 +2,12 @@
 
 stdenv.mkDerivation rec {
   pname    = "elastix";
-  pversion = "4.9.0";
+  pversion = "5.0.0";
   name  = "${pname}-${pversion}";
 
   src = fetchurl {
     url    = "https://github.com/SuperElastix/${pname}/archive/${pversion}.tar.gz";
```
Comment (c0bw3b) [inline pkgs/development/libraries/science/biology/elastix/default.nix]: Don't use fetchurl on GH _archives_ as they get regenerated from time to time and the hash may change.
You should use `fetchFromGitHub` or  `fetchzip` instead.

File: pkgs/development/libraries/science/biology/elastix/default.nix [hunk 2]
```diff
@@ -2,12 +2,12 @@
 
 stdenv.mkDerivation rec {
   pname    = "elastix";
-  pversion = "4.9.0";
+  pversion = "5.0.0";
   name  = "${pname}-${pversion}";
```
Comment (c0bw3b) [inline pkgs/development/libraries/science/biology/elastix/default.nix]: Just
```nix
    pname = "elastix";
    version = "5.0.0";
```
is enough (`name` will be defined automatically)

**Review by c0bw3b — CHANGES_REQUESTED** [review]

File: pkgs/development/libraries/science/biology/elastix/default.nix (hunk 1 above)
Comment (bcdarwin) [reply pkgs/development/libraries/science/biology/elastix/default.nix]: Good catch, thanks.

File: pkgs/development/libraries/science/biology/elastix/default.nix (hunk 2 above)
Comment (bcdarwin) [reply pkgs/development/libraries/science/biology/elastix/default.nix]: Updated.

File: pkgs/development/libraries/science/biology/elastix/default.nix [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
-  name  = "${pname}-${pversion}";
+  version = "5.0.0";
 
-  src = fetchurl {
-    url    = "https://github.com/SuperElastix/${pname}/archive/${pversion}.tar.gz";
-    sha256 = "02pbln36nq98xxfyqwlxg7b6gmigdq4fgfqr9mym1qn58aj04shg";
+  src = fetchFromGitHub {
+    owner  = "SuperElastix";
+    repo   = pname;
+    rev    = "4c1441bcbc29782b5cbf100a40a433e6689d5ffe";
```
Comment (c0bw3b) [inline pkgs/development/libraries/science/biology/elastix/default.nix]:
```suggestion
    rev    = version;
```
to fetch upstream tag named after the declared version.

It makes the link between source and version clearer, plus it allows auto-updates of the pkg by nixpkgs-update.

Comment (bcdarwin) [reply pkgs/development/libraries/science/biology/elastix/default.nix]: Done.

**Review by c0bw3b — APPROVED** [review]
Builds and runs

Result of `nix-review pr 71710` [1](https://github.com/Mic92/nix-review)
<details>
  <summary>1 package were build:</summary>

  - elastix
</details>

**Pull request merged**